### DISCUSSION TITLE
what are the technical restrictions of localsend?

### DISCUSSION THREAD

**Discussion by ghost** [localsend/localsend#191] [Discussion (English, International)] [answered]
Hi.

what are the technical restrictions of localsend? - for example, is there any file size limit when sending stuff in localsend?

**Comment by Tienisto** [comment, marked as answer]
Technically, there is no limitation in the protocol. I have seen someone transferring multiple GBs. But some users have issues with large files though #36

**Reply by ghost** [reply]
thanks for comment!
